src/experimental/reputation/types.ts

```diff
@@ -25,13 +29,41 @@ export interface ReputationProviderEntry {
   scores: ReputationProviderScores;
 }
 
+/**
+ * Information about a rejected operator.
+ */
+export interface ReputationRejectedOperator {
+  wallet: string;
```

Comment:
For consistency, please note that we have the `ProviderInfo` type available. It contains `id`, `name` and `WalletAddress`. Perhaps you could use that type here as well.